pkg/e2e/e2e.go

```diff
@@ -751,6 +751,15 @@ func runTestsInPhase(
 		}
 	}
 
+	// Generate JUnit report once all tests have finished with customized settings
+	ginkgo.ReportAfterSuite("OSDE2E", func(report ginkgo.Report) {
+		reporters.GenerateJUnitReportWithConfig(
```

Comment:
need to check err https://cs.github.com/onsi/ginkgo/blob/880485988b4b99c965b2192865e2bbe69a58dd01/reporters/junit_report_test.go?q=reporters.GenerateJUnitReportWithConfig#L218-L223